.ebextensions/nginx.config

```diff
@@ -332,31 +332,6 @@ files:
 
         charset utf-8;
 
-        # With Content Security Policy (CSP) enabled (and a browser that supports it (http://caniuse.com/#feat=contentsecuritypolicy),
-        # you can tell the browser that it can only download content from the domains you explicitly allow
-        # CSP can be quite difficult to configure, and cause real issues if you get it wrong
-        # There is website that helps you generate a policy here http://cspisawesome.com/
-        add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' https://dap.digitalgov.gov https://www.google-analytics.com; style-src 'self' 'unsafe-inline'; img-src 'self' data: https://www.google-analytics.com *.tile.openstreetmap.org; frame-ancestors 'none'; frame-src https://www.youtube.com;" always;
```

Comment:
We only need to make sure this CSP header is delivered on root requests.  For the error pages in particular, as well as any other directly delivered HTML, we should add one in.